### DISCUSSION TITLE
a标签的href属性使用环境变量site.url和site.postUrl不生效

### DISCUSSION THREAD

**Discussion by liyitongxue** [walinejs/waline#1234] [Q&A] [answered]
烦请各位大佬解答，前端知识比较薄弱， 谢谢抽出时间回答，不胜感激。

问题为标题，详细介绍如下：
目前仅配置了回复评论的邮件通知功能。使用的部署方式为Leancloud+Vercel，且已在Vercel的环境变量中配置并重新部署，如下图所示：
![图片](https://user-images.githubusercontent.com/48759692/181290019-ec2a4e3d-4707-45d6-9395-314f6e16768c.png)

MAIL_TEMPLATE的通知配置如下所示：
![图片](https://user-images.githubusercontent.com/48759692/181291069-b3b08ef2-2aa5-4826-b7a6-05ca4a06929a.png)
 
[博客评论地址https://liyitongxue.github.io/perception/](https://liyitongxue.github.io/perception/)，评论截图如下：
![图片](https://user-images.githubusercontent.com/48759692/181292402-1f9d56c6-6029-479d-8013-5bd80940984b.png)

然而收到的邮件如下图所示，【站点名】和【查看完整内容】href属性无值传入。
![图片](https://user-images.githubusercontent.com/48759692/181293441-0176d0f3-36c9-409d-a475-d9f5c968082e.png)

**Comment by lizheming** [comment, marked as answer]
SITE_URL 加一下 `https://` 前缀

**Reply by liyitongxue** [reply]
加了http前缀，问题已解决，谢谢大佬！
